pkg/storaged/index.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <!--
- * This file is part of Cockpit.
+ * This file is part of CockPWNED
```

Comment:
I don't like this change.  I'd prefer if we didn't proceed with it.

Reply:
Damn, someone noticed my 31337 h4x0r attempt! :see_no_evil: